sdk/dotnet/Redis/Instance.cs

```diff
@@ -330,12 +330,6 @@ public sealed class InstanceArgs : Pulumi.ResourceArgs
         [Input("authEnabled")]
         public Input<bool>? AuthEnabled { get; set; }
 
-        /// <summary>
-        /// AUTH String set on the instance. This field will only be populated if auth_enabled is true.
-        /// </summary>
-        [Input("authString")]
-        public Input<string>? AuthString { get; set; }
```

Comment:
Breaking change?

Reply:
This is expected - this was never a valid input - it's a computed field only